corehq/apps/domain/views/accounting.py

```diff
@@ -1270,27 +1270,12 @@ class SelectedAnnualPlanView(ContactFormViewBase):
     request_type = 'Annual Plan Request'
 
     @property
-    def on_annual_plan(self):
```

Comment:
Can you explain what does this view do and link the code that shows this view is "only accessible when already on a Pay Annually plan"